chatterbot/storage/django_storage.py

```diff
@@ -138,6 +138,21 @@ def remove(self, statement_text):
         responses.delete()
         statements.delete()
 
+    def get_latest_response(self, conversation_id):
+        """
+        Returns the latest response in a conversation if it exists.
+        Returns None if a matching conversation cannot be found.
+        """
+        from django.apps import apps
```

Comment:
``from django.apps import apps`` are many places, instead can we import one time on top of the page?